src/python/TaskWorker/Actions/DBSDataDiscovery.py

```diff
@@ -92,7 +89,7 @@ def checkBlocksSize(self, blocks):
                 msg += "\nhttps://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3FAQ"
                 raise TaskWorkerException(msg)
 
-    def requestTapeRecall(self, blockList=[], system='Dynamo', msgHead=''):   # pylint: disable=W0102
+    def requestTapeRecall(self, blockList, sizeToRecall, system='Dynamo', msgHead=''):   # pylint: disable=W0102
         """
         :param blockList: a list of blocks to recall from Tape to Disk
```

Comment:
add a description of new sizeToRecall param
Also time to change the default for system from Dynamo to Rucio